ansible/vault_nomad_integration.yml

```diff
@@ -0,0 +1,57 @@
+- name: Start vault's nomad secrets backend
+  shell:
+    cmd: vault secrets enable nomad
+  environment:
+    VAULT_ADDR: http://127.0.0.1:8200
+    VAULT_TOKEN: master
+
+- name: fetch bootstrap token nomad
+  shell:
+    cmd: vault kv get -field=secret-id secret/nomad-bootstrap-token
+  register: bootstrap
```

Comment:
```suggestion
  register: bootstrap_token
```

Reply:
That'll require refactoring a few other places as well, but I agree, more verbose. I'll fix it!

Reply:
Fixed